explainability-service/manifests/opendatahub/default/trustyai-deployment.yaml

```diff
@@ -24,7 +24,6 @@ spec:
     app.kubernetes.io/name: trustyai-service
     app.kubernetes.io/version: 999-SNAPSHOT
   type: ClusterIP
-
 ---
 apiVersion: image.openshift.io/v1
```

Comment:
I would vote to delete this image stream

Reply:
Under the "containers" key in a deployment you can also specify an image, like so:
```
image: quay.io/ruimvieira/trustyai-service:<tag>
```

We should do that.
Related... should we move the image over to the trusyai org?

Reply:
@anishasthana Good point. We're moving it to https://quay.io/organization/trustyai